### PR TITLE
feature(core): AutoEnable GuardDuty S3 Protection

### PR DESCRIPTION
**Pull request by naveenkoppula**
#478 #480 
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Brian969 — DISMISSED** [review]
LGTM - Made a few tweaks.

File: src/deployments/cdk/src/deployments/iam/guardduty-roles.ts [hunk 1]
```diff
@@ -73,7 +73,8 @@ export async function createAdminSetupRole(stack: AccountStack) {
         'guardduty:CreateMembers',
         'guardduty:UpdateOrganizationConfiguration',
         'guardduty:DescribeOrganizationConfiguration',
-        'guardduty:updateMemberDetectors',
+        'guardduty:UpdateMemberDetectors',
```
Comment (rjjaegeraws) [inline src/deployments/cdk/src/deployments/iam/guardduty-roles.ts:76]: I don't see "UpdateMemberDetectors" as a valid permission for GuardDuty.
image

Comment (Brian969) [reply src/deployments/cdk/src/deployments/iam/guardduty-roles.ts:76]: @naveenkoppula - should this be UpdateDetector?

**Review by rjjaegeraws — APPROVED** [review]
Looks good to me.

File: src/deployments/cdk/src/deployments/iam/guardduty-roles.ts (hunk 1 above)
Comment (Brian969) [reply src/deployments/cdk/src/deployments/iam/guardduty-roles.ts:76]: After testing, "UpdateMemberDetectors" is the required permission.

**Review by Brian969 — APPROVED** [review]
Looks Good.

**Pull request merged**